Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -13,8 +13,14 @@
 
 #include <cctype>
 #include <cmath>
+#include <optional>
```

Comment:
Was this intended to be left included? No part of the changes seem to use std::optional

Reply:
Nope, remnant from abandoned design.